packages/execution-environments/webpack.config.js

```diff
@@ -115,6 +115,10 @@ module.exports = (_, argv) => {
         // Without this alias webpack tried to require ../../node_modules/stream/ which doesn't have Duplex, breaking the bundle
         stream: 'stream-browserify',
       },
+      fallback: {
```

Comment:
Why is this needed now?

Reply:
I think Webpack tries to load `node-fetch` and `ws` since it's `require()`'d, even though not used in the browser builds.